### PR TITLE
Full react web application

### PR DESCRIPTION
**Pull request by AjroudiRida**
In this featured branch, I did the following:
Select an API.
Build files structure for React app.
create routes and navigation in the app.
Style components
Deploy the project.
Record a video for the project.
Create a README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give me star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank My Coding Partner.
```
Comment (abdulhamiid) [inline README.md]: - [ ] According to the project requirements, our README should contain an "Acknowledgements" section giving appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 😊

File: README.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo](https://65029051474b2c132566a13d--velvety-hamster-d0b4f7.netlify.app/)
+
+
+ ## 🚀 Video Presentation <a name="video"></a>
+
+- [Video](https://www.loom.com/share/b8b4dace106a40d18b1642a9e188af32?sid=08d83a11-8e9a-42fe-8495-506219f15e55)
+
```
Comment (abdulhamiid) [inline README.md]: Good job on your React capstone project, you have really done well 👍

*Feedback about your video presentation.*

- [ ] In your [video presentation](https://www.loom.com/share/b8b4dace106a40d18b1642a9e188af32?sid=08d83a11-8e9a-42fe-8495-506219f15e55), you didn't highlight the piece of the code that you find interesting or proud of making. 🤓

According to the project requirements, The content of the video should include:

1. The student recorded a video presenting the features of the project. ✅ 
2. The maximum length of the video is 5 minutes. ✅ 
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear. ✅ 
4. The video contains a demo of the project, highlighting all the project features. ✅ 
5. The video contains a highlight about a piece of the code the student finds interesting or is proud of making. ❌ 


I would recommend that you record a new video that includes all 5 details as requested in the project requirement 🛠👍

File: src/components/header/__test__/Header.test.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  it('shoud be in the document', () => {
+    render(<Header />);
+    expect(screen.getByTestId('header')).toBeInTheDocument();
+  });
+
+  it('shoud be visible in the document', () => {
+    render(<Header />);
+    expect(screen.getByTestId('header')).toBeVisible();
+  });
+});
```
Comment (abdulhamiid) [inline src/components/header/__test__/Header.test.js]: **Testing**
image
image
image

- [ ] Please ensure that all the tests are passing, The main objective of unit testing is to isolate written code to test and determine if it works as intended which is an important step in the development process. 🛠😊

- [ ] If done correctly, unit tests can detect early flaws in code which may be more difficult to find in later testing stages.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+- Make the Fruit Tree responsive
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](http://ridaajroud.me/Bookstore/issues).
```
Comment (abdulhamiid) [inline README.md]: The link to the `issues` page is broken, kindly fix that. Let's avoid broken links in our readme for a professional repository. 😊

image

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
Hi @AjroudiRida 👋, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by ShadyShawkat — APPROVED** [review]
## Hi @AjroudiRida, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**